app/models/event_invoice.py

```diff
@@ -86,6 +87,14 @@ def previous_month_date(self):
     def due_at(self):
         return self.issued_at + timedelta(days=EventInvoice.DUE_DATE_DAYS)
 
+    @property
+    def issued_at_tz(self):
+        return self.issued_at.astimezone(pytz.timezone(self.event.timezone))
+
+    @property
+    def due_at_tz(self):
```

Comment:
Please use datetime helper, it has timezone support already

Reply:
@iamareebjamal `return self.due_at.astimezone(datetime.astimezone(self.event.timezone))` like this?

Reply:
Or you meant `datetime(order.event.timezone)` like this in jinja template?